.github/workflows/pr-cpu.yaml

```diff
@@ -16,10 +16,6 @@ jobs:
     strategy:
       matrix:
         include:
-        - name: cpu-3.11-2.2
```

Comment:
you will need to similarly remove GPU jobs for 2.2

Reply:
Yes, in progress